### PR TITLE
SharedHost always installs to 'ProgramFiles'\dotnet; 3.0

### PR DESCRIPTION
**Pull request by ghost**
When installing the SharedHost; copy all files to the traditional default install location: 'ProgramFiles'\dotnet even when installing somewhere else.

This is part of the work to allow SDKs, when installing with VS, to install to a different volume when VS is doing the same.

**User Scenerio:**
When a user has installed a 2.1 SDK or similar SDK first so that the PATH environmental variable has 'ProgramFiles'\dotnet in a first position. Then the user installs a 3.0 SDK thru VS to a different volume with VS so that 'VSSharedVolume'\dotnet is in a second position. We want the 3.0 SharedHost to be on the PATH in the first position. Therefore, installing the SharedHost to both 'VSSharedVolume'\dotnet and 'ProgramFiles'\dotnet will achieve this.

Reference: dotnet/core-sdk#2174
Similar: https://github.com/dotnet/core-setup/pull/7658

@dotnet/dotnet-cli

### CODE REVIEW

File: src/pkg/packaging/windows/host/host.wxs
Comment (dagood) [inline src/pkg/packaging/windows/host/host.wxs]: Could you leave this the same as it is in `master` to avoid unnecessary differences?

https://github.com/dotnet/core-setup/blob/8a5f710c474a955ae82ce449fb9185a781782558/src/pkg/projects/Microsoft.NETCore.DotNetHost/host.wxs#L10-L21

**Review by dagood — APPROVED** [review]
https://github.com/dotnet/core-setup/pull/7764 is an in-progress PR porting the new shiny infra, which is why the files are in currently in a different place with more duplication. The impl in this PR looks good. Let's merge this PR before https://github.com/dotnet/core-setup/pull/7764 once they have approval, I'll deal with merge conflicts over there.

File: src/pkg/packaging/windows/host/host.wxs
Comment (johnbeisner) [reply src/pkg/packaging/windows/host/host.wxs]: Done.

**Comment by johnbeisner** [comment]
**Description**
When installing the SharedHost; copy all files to the traditional default install location: 'ProgramFiles'\dotnet even when installing somewhere else. This is part of the work to allow SDKs, when installing with VS, to install to a different volume when VS is doing the same.

**Customer Impact**
When a user installs 16.3 Visual Studio with the 3.0 SDK to a non-operating system volume, the 3.0 SharedHost will also install to 'ProgramFiles'\dotnet

**Regression**
No.

**Risk**
Low.

**Comment by johnbeisner** [comment]
image

**Pull request merged**
